### PR TITLE
Feat: Update create profile method

### PR DESCRIPTION
**Pull request by carlosmondra**
This PR updates the method used to create profiles using the super user

Needs https://github.com/openreview/openreview-py/pull/2447 to be merged first

### CODE REVIEW

File: openreview/tools.py [hunk 1]
```diff
@@ -343,7 +343,7 @@ def get_invitation(client, id):
         print('Can not retrieve invitation', e)
     return invitation
 
-def create_profile(client, email, fullname, url='http://no_url', allow_duplicates=False):
+def create_profile(client, email, fullname, super_user='openreview.net'):
```
Comment (celestemartinez) [inline openreview/tools.py:346]: Will super_user always be `openreview.net`? Won't it be `OpenReview.net` on live?

Comment (carlosmondra) [reply openreview/tools.py:346]: Yes, it will be. I wasn't sure how else to pass it because that information is in the config file of the API.

Comment (celestemartinez) [reply openreview/tools.py:346]: I see. We could do something like this: https://github.com/openreview/openreview-py/blob/03d796c40e5bb950d3e716606f6bdaf05c308b8d/openreview/tools.py#L135 but if we are not using this in any process and we only do it manually, then it shouldn't be an issue

Comment (carlosmondra) [reply openreview/tools.py:346]: Do you mean something like this?

```py
            super_user = 'openreview.net'
            if 'https://devapi' in client.baseurl or 'https://api' in client.baseurl:
                super_user =  'OpenReview.net'
```

Comment (carlosmondra) [reply openreview/tools.py:346]: > but if we are not using this in any process and we only do it manually, then it shouldn't be an issue
Yeah, I don't see how we would use this in a process function

**Review by celestemartinez — APPROVED** [review]

**Pull request merged**